routes/donations.js

```diff
@@ -110,6 +113,9 @@ router.post('/', async (req, res) => {
         submitted_date, last_edited_date, status
         )
       VALUES (
+        $(id),
+        ${routeId ? '$(routeId), ' : ''}
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'routeId' is not defined.